### ISSUE TITLE
Training accuracy suddenly approaches zero

### ISSUE THREAD

**Issue by liwei109** [facebookresearch/AlphaNet#9]
Hi, I meet a question during the training process, the accuracy suddenly approaches zero (just like the network parameters are re-initialized). The log is as follows:

Epoch: [48][ 70/625]	Time  1.383 ( 1.428)	Data  0.000 ( 0.152)	Loss -1.8897e+00 (-2.0853e+00)	Acc@1  56.54 ( 60.11)	Acc@5  79.59 ( 81.28)
Epoch: [48][ 80/625]	Time  1.346 ( 1.401)	Data  0.000 ( 0.133)	Loss -2.3125e+00 (-2.0805e+00)	Acc@1  57.71 ( 60.05)	Acc@5  79.44 ( 81.26)
Epoch: [48][ 90/625]	Time  1.224 ( 1.386)	Data  0.000 ( 0.119)	Loss -2.1656e+00 (-2.0629e+00)	Acc@1  58.15 ( 59.98)	Acc@5  80.13 ( 81.25)
Epoch: [48][100/625]	Time  1.474 ( 1.375)	Data  0.000 ( 0.107)	Loss -2.3581e+00 (-2.0779e+00)	Acc@1  58.06 ( 59.92)	Acc@5  78.66 ( 81.22)
Epoch: [48][110/625]	Time  1.215 ( 1.367)	Data  0.000 ( 0.097)	Loss -4.3465e+00 (-2.1967e+00)	Acc@1   0.10 ( 56.28)	Acc@5   0.59 ( 76.59)
Epoch: [48][120/625]	Time  1.111 ( 1.357)	Data  0.000 ( 0.089)	Loss -4.5741e+00 (-2.3879e+00)	Acc@1   0.15 ( 51.64)	Acc@5   0.39 ( 70.31)
Epoch: [48][130/625]	Time  1.152 ( 1.350)	Data  0.000 ( 0.083)	Loss -4.5668e+00 (-2.5548e+00)	Acc@1   0.05 ( 47.71)	Acc@5   0.44 ( 64.99)
Epoch: [48][140/625]	Time  1.232 ( 1.346)	Data  0.000 ( 0.077)	Loss -4.5077e+00 (-2.6952e+00)	Acc@1   0.29 ( 44.34)	Acc@5   0.88 ( 60.44)
Epoch: [48][150/625]	Time  1.219 ( 1.341)	Data  0.000 ( 0.072)	Loss -4.4784e+00 (-2.8117e+00)	Acc@1   0.34 ( 41.41)	Acc@5   1.12 ( 56.49)
Epoch: [48][160/625]	Time  1.274 ( 1.338)	Data  0.000 ( 0.067)	Loss -4.2812e+00 (-2.9086e+00)	Acc@1   0.34 ( 38.86)	Acc@5   0.93 ( 53.04)

What's more, the training loss is always negative, is that right?

**Comment by liwei9719** [comment]
..

**Comment by dilinwang820** [comment]
Hi @liwei9719,

Thanks for your interest in this repo. Does this happen all the time? Would you be able to load the previous checkpoint and resume the training with expected performance? From my experiences, the training is largely stable if the gradients are clipped properly (e.g., set clip_grad_val to 1.0)

Regarding the negative training loss, I must apologize for the confusion here. The reason is that we don't use the actual alpha-divergence for logging. Instead we first compute the gradient we need, and then constructing a surrogate loss to produce this gradient. The surrogate loss might be negative. Thanks.

**Comment by liwei109** [comment]
I only trained once because of the huge cost.I have set clip_grad_val to 1.0. Does this parameter setting cause the above phenomenon?

**Comment by dilinwang820** [comment]
Maybe try to resume from a previously saved checkpoint with a different random seed?

**Comment by jun-fang** [comment]
Hi @liwei9719 @dilinwang820, 

I met the same problem at around 60 epoch, and I tried multiple times to resume from a saved checkpoint with another random seed but it did not solve the issue. 

@liwei9719 did you find a solution for this?

@dilinwang820 do you have an idea why this happens and what could be a good way to avoid this phenomenon?

Thanks!

**Comment by dilinwang820** [comment]
@jun-fang to my experience, the training is always stable with the default settings; maybe try to warm up a little bit with less regularization and data augmentation, and then resume with default settings?

**Comment by pprp** [comment]
I met the same problem.. 

![image](https://user-images.githubusercontent.com/29230784/166613874-9d21d06f-0b77-4b71-9e43-150b5dbaa670.png)
